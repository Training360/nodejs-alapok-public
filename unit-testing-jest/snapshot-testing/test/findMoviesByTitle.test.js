const findMoviesByTitle = require('../src/findMoviesByTitle')

test('findMoviesByTitle should be an array of movie objects', () => {
//   expect(findMoviesByTitle('F')).toEqual([
//     { title: 'Flipper', emoji: '🐬' }
//   ])
  expect.addSnapshotSerializer({
    test: ({ title, emoji }) => title && emoji,
    print: ({ title, emoji }) => `${emoji} ${title}`
  })
  expect(findMoviesByTitle('F')).toMatchSnapshot()
})
