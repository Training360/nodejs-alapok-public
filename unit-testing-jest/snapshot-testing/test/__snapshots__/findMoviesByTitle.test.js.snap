// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`findMoviesByTitle should be an array of movie objects 1`] = `
Array [
  🐬 Flipper,
]
`;
